.github/workflows/docker_build_and_publish.yml

```diff
@@ -0,0 +1,92 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build and Publish Docker
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: 
+      - master
+      - ra_master_docker_build_n_publish
+  pull_request:
+    branches: 
+      - master
+      - ra_master_docker_build_n_publish
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: Get Branch Name
+      id: get_branch
+      shell: bash
+      run: echo "##[set-output name=BRANCH_NAME;]$(echo ${GITHUB_HEAD_REF#refs/heads/})"
+
+    - name: Install Clojure tools
+      uses: DeLaGuardo/setup-clojure@2.0
+      with:
+        leiningen: 2.9.1
+        boot: 2.8.3
+        tools-deps: 1.10.1.469
```

Comment:
Do we need leiningen and tools-deps?

Reply:
I'm not sure. I can look.

Reply:
It appears that we do not need those - I removed them both.